contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -386,20 +369,22 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
         bytes32 failedMessageHash = receivedFailedMessageHashes[originChainID][
             message.messageID
         ];
-        if (failedMessageHash == bytes32(0)) {
-            revert MessageNotFound();
-        }
-
-        if (keccak256(abi.encode(message)) != failedMessageHash) {
-            revert InvalidMessageHash();
-        }
+        require(
+            failedMessageHash != bytes32(0),
+            "TeleporterMessenger: message not found"
+        );
+        require(
+            keccak256(abi.encode(message)) == failedMessageHash,
+            "TeleporterMessenger: invalid message hash"
+        );
 
         // Check that the target address has fully initialized contract code prior to calling it.
         // If the target address does not have code, the execution automatically fails because
         // we disallow calling EOA addresses.
-        if (message.destinationAddress.code.length == 0) {
-            revert InvalidDestinationAddress();
-        }
+        require(
+            message.destinationAddress.code.length > 0,
```

Comment:
Is length guaranteed to be a uint? Even if it's an int, I assume a negative value would indicate some error.

Reply:
length is returned as `uint` or `uint256`, so cannot be negative.